Enable auto-apply for run trigger runs (#377)
hashicorp/terraform-docs-common: website/docs/cloud-docs/workspaces/settings/index.mdx

To minimize the number of runs that error when changing your workspace's execution mode, you should:

1. Disable [auto-apply](/terraform/cloud-docs/workspaces/settings#auto-apply) if you have it enabled.
1. Complete any runs that are no longer in the [pending stage](/terraform/cloud-docs/run/states#the-pending-stage).
1. [Lock](/terraform/cloud-docs/workspaces/settings#locking) your workspace to prevent any new runs.
1. Change the execution mode.
1. Enable [auto-apply](/terraform/cloud-docs/workspaces/settings#auto-apply), if you had it enabled before changing your execution mode.
1. [Unlock](/terraform/cloud-docs/workspaces/settings#locking) your workspace.

<a id="apply-method"></a>
<a id="auto-apply-and-manual-apply"></a>

### Auto-apply

Whether or not Terraform Cloud should automatically apply a successful Terraform plan. If you choose manual apply, an operator must confirm a successful plan and choose to apply it.

<!-- BEGIN: TFC:only name:auto-apply-run-trigger -->

> In addition to the main auto-apply setting (which affects runs created by the UI, API, CLI, and version control webhooks), Terraform Cloud also has a separate setting for runs created by [run triggers](/terraform/cloud-docs/workspaces/settings/run-triggers) from another workspace.

Reviewer: I think this flows slightly better as two sentences:
```suggestion
The main auto-apply setting affects runs created by the Terraform Cloud user interface, API, CLI, and version control webhooks. Terraform Cloud also has a separate setting for runs created by [run triggers](/terraform/cloud-docs/workspaces/settings/run-triggers) from another workspace.
```
